### ISSUE TITLE
Add flux-command for easily finding path for certain value

### ISSUE THREAD

**Issue by TobiasNx** [metafacture/metafacture-fix#347]
Add to the inspection commands: `list-fix-path` and `list-fix-values`
a flux command for identifying a path to a certain value would be great without hassling with triples:

```
| fix("nothing()",repeatedFieldsToEntities="true")
| flatten
| stream-to-triples
| filter-triples(objectpattern="[value]")
| template("${p}\t ${o}")
```

a macro command like `find-fix-path("[value]")` would be nice.

[See example in playground.](https://metafacture.org/playground/?flux=inputFile+//+This+is+a+variable+for+the+input+data.%0A%0A//+These+are+Flux+commands.%0A%7C+open-file%0A%7C+as-records%0A%7C+decode-json%0A%7C+fix%28%22nothing%28%29%22%2CrepeatedFieldsToEntities%3D%22true%22%29%0A%7C+flatten%0A%7C+stream-to-triples%0A%7C+filter-triples%28objectpattern%3D%22example%22%29%0A%7C+template%28%22%24%7Bp%7D\t+%24%7Bo%7D%22%29%0A%7C+print%0A%3B%0A&data=%7B%0A++++%22Hello%22%3A+%22World%22%2C%0A++++%22This%22%3A+%5B%22is%22%2C%22an%22%2C%22input%22%2C%22example%22%5D%2C%0A++++%22Please%22%3A+%22Paste+your+input+data+here%2C+if+needed%22%0A%7D)

**Comment by TobiasNx** [comment]
https://metafacture.org/playground/?flux=%22https%3A//d-nb.info/gnd/1160720738/about/marcxml%22%0A%7C+open-http%28accept%3D%22application/xml%22%29%0A%7C+decode-xml%0A%7C+handle-marcxml%0A%7C+find-fix-paths%28%22.%2AETL.%2A%22%29%0A%7C+print%0A%3B


Seems that the template is not right yet.

**Comment by TobiasNx** [comment]
Should be fixed with https://github.com/metafacture/metafacture-fix/pull/364
